src/entrypoints/main/index.html

```diff
@@ -84,15 +85,20 @@
     </style>
   </head>
   <body>
-      <a
-        id='skip-to-content'
-        class='screenreader-text'
-        href='#content'
-        aria-hidden='false'
-        lang='en'
-      >
-        Skip to Content
-      </a>
+    <a
+      id="skip-to-content"
+      class="screenreader-text"
+      href="#content"
+      aria-hidden="false"
+      lang="en"
+      onclick="
+          event.preventDefault();
+          event.stopPropagation();
+          document.getElementById('content').scrollIntoView();
```

Comment:
Mulig denne kan forenkles til bare:

```suggestion
window.location.hash = '#content';
```

Reply:
OK. Skal teste det ut! 👍

Reply:
Den tvinger en nedlasting av tabellen. Tenker det er lurt å beholde nåværende løsning.